service/src/server/routes/cost.rs

```diff
@@ -0,0 +1,16 @@
+// Copyright 2023-, GraphOps and Semiotic Labs.
+// SPDX-License-Identifier: Apache-2.0
+
+use async_graphql_axum::{GraphQLRequest, GraphQLResponse};
+use axum::extract::Extension;
+
+use crate::{common::indexer_management_client::schema::CostSchema, server::ServerOptions};
+
+pub(crate) async fn graphql_handler(
+    req: GraphQLRequest,
+    Extension(schema): Extension<CostSchema>,
+    Extension(server_options): Extension<ServerOptions>,
+) -> GraphQLResponse {
+    let response = async move { schema.execute(req.into_inner().data(server_options)).await }.await;
+    response.into()
```

Comment:
That should be equivalent
```suggestion
    schema.execute(req.into_inner().data(server_options)).await.into()
```